srcCppCommon/ParameterManager.cxx

```diff
@@ -406,7 +406,7 @@ void ParameterManager::initialize()
             "(Disabled for LatencyTest mode or if dataLen > 4kB)");
     batchSize->set_type(T_NUMERIC_LD);
     batchSize->set_extra_argument(YES);
-    batchSize->set_range(0, MAX_SYNCHRONOUS_SIZE - 1);
+    batchSize->set_range(0, MAX_PERFTEST_SAMPLE_SIZE - 1);
```

Comment:
We had a dependency with MAX_SYNCRHONOUS_SIZE here, however, as at the point where we create the manager we know nothing about the properties set, I am setting the max size of the batch as the max size of a sample.